tests/test_dataset.py

```diff
@@ -93,6 +93,24 @@ def test_german_traffic_sign(self):
                 self.assertTrue(25 <= x.shape[0] <= 232)
                 self.assertTrue(25 <= x.shape[1] <= 266)
 
+    def test_resisc45(self):
+        for split, size in [("train", 22500), ("validation", 4500), ("test", 4500)]:
```

Comment:
So adding this added 8min to test time since it has to be downloaded. Perhaps we add a SkipTest if the dataset isn't available? That way we could test locally if we want. Part of a solution for #238

Reply:
Added a skip